payments-core/src/test/java/com/stripe/android/ConfirmSetupIntentParamsFactoryTest.kt

```diff
@@ -0,0 +1,38 @@
+package com.stripe.android
+
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.model.ConfirmSetupIntentParams
+import com.stripe.android.model.ConfirmStripeIntentParams
+import com.stripe.android.model.MandateDataParams
+import com.stripe.android.testing.PaymentMethodFactory
+import com.stripe.android.testing.SetupIntentFactory
+import org.junit.Test
+
+class ConfirmSetupIntentParamsFactoryTest {
+    @Test
+    fun `create() should not contain mandate data`() {
```

Comment:
```suggestion
    fun `create() should contain mandate data`() {
```
I think this was just a typo